metadata-ingestion/src/datahub/ingestion/source/looker/looker_source.py

```diff
@@ -167,23 +198,26 @@ class LookerDashboardSource(TestableSource):
     :::
     """
 
+    platform = "looker"
     source_config: LookerDashboardSourceConfig
     reporter: LookerDashboardSourceReport
     user_registry: LookerUserRegistry
-    explores_to_fetch_set: Dict[Tuple[str, str], List[str]] = {}
     accessed_dashboards: int = 0
     resolved_user_ids: int = 0
     email_ids_missing: int = 0  # resolved users with missing email addresses
 
     def __init__(self, config: LookerDashboardSourceConfig, ctx: PipelineContext):
-        super().__init__(ctx)
+        super().__init__(config, ctx)
         self.source_config = config
         self.reporter = LookerDashboardSourceReport()
         self.looker_api: LookerAPI = LookerAPI(self.source_config)
         self.user_registry = LookerUserRegistry(self.looker_api)
         self.explore_registry = LookerExploreRegistry(self.looker_api, self.reporter)
         self.reporter._looker_explore_registry = self.explore_registry
         self.reporter._looker_api = self.looker_api
+
+        self.explores_to_fetch_set: Dict[Tuple[str, str], List[str]] = {}
```

Comment:
why was this necessary?

Reply:
This is not necessary if every ingestion (pipeline) runs in separate process. However if two pipelines are running in same process, it is observed that `explores_to_fetch_set` from earlier pipeline persisted for next pipeline run, leading to incorrect results. Noticed this when adding tests for stateful ingestion.

Reply:
ah that's pretty subtle - we should watch out for those sorts of bugs in our other sources, since we definitely do the same thing elsewhere